components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -7412,8 +7412,8 @@ def getWindowMin(self):
                            PixelsTypeuint16: 0,
                            PixelsTypeint32: -32768,
                            PixelsTypeuint32: 0,
-                           PixelsTypefloat: -32768,
-                           PixelsTypedouble: -32768}
+                           PixelsTypefloat: -2147483648,
```

Comment:
problem for pixelstype signed/unsigned 32
The value should not be hardcoded in the file but using the methods ``getPixelsTypeLowerBound/UpperBound`` from the RE